ch_pipeline/core/dataquery.py

```diff
@@ -49,12 +49,12 @@
 _DEFAULT_NODE_SPOOF = {"cedar_online": "/project/rpp-krs/chime/chime_online/"}
 
 
-def _force_list(val):
+def _force_list(val) -> list:
     """Ensure configuration property is a list."""
     if val is None:
         return []
     elif hasattr(val, "__iter__"):
```

Comment:
```suggestion
    elif hasattr(val, "__iter__") and not isinstance(val, str):
```
This obviously isn't your problem, but I think this code predates our Python 3 transition (when `str` did not have an `__iter__` method), and on Python 3 it will turn a string into an array of the individual characters.